regulations/views/comment.py

```diff
@@ -23,6 +31,7 @@ def upload_proxy(request):
     url = s3.generate_presigned_url(
         ClientMethod='put_object',
         Params={
+            'ContentLength': size,
```

Comment:
I didn't see documentation for this, but based on experimentation, but uploading a file that was too big errored with `SIGPIPE`, and uploading a file that was too small error with a signature mismatch. I'll check whether the file actually got uploaded in the first case.

And I don't know of any special values for `Content-Length`--if we don't want to validate, I think we just skip that parameter when generating the signed URL.